### PR TITLE
✅ - Add tests for update SPARQL queries. Fix #91

### LINKED ISSUE #91: More SPARQL Update Tests
Flesh out more tests to exercise the SPARQL update statements.

### PR DESCRIPTION
**Pull request by snowell**

### CODE REVIEW

File: test/query.spec.js [hunk 1]
```diff
@@ -195,42 +195,50 @@ describe('query.execute()', () => {
   });
 
   describe('update', () => {
-    it('should insert some data', () =>
-      execute(`insert data {:foo :bar :baz}`)
+    let clearDatabase = () => execute('clear all');
```
Comment (arb) [inline test/query.spec.js]: `const`. Also "clear all" is a real query?

Comment (arb) [inline test/query.spec.js]: Actually you don't even need to make this a variable, just do `afterEach(() => execue('clear all'))`

File: test/query.spec.js [hunk 2]
```diff
@@ -195,42 +195,50 @@ describe('query.execute()', () => {
...
+    afterEach(clearDatabase);
+
+    it('should support "insert data"', () => {
+      let data = '{:foo :bar :baz}';
```
Comment (arb) [inline test/query.spec.js]: `const` or just inline it with the query.

File: test/query.spec.js [hunk 3]
```diff
@@ -195,42 +195,50 @@ describe('query.execute()', () => {
...
           expect(res.body.results.bindings).toHaveLength(1);
-        }));
+        });
+    });
 
-    it('should remove some data', () =>
-      execute(`select * {<urn:foo> ?p ?o}`)
+    it('should support "delete data"', () => {
+      let data = '<urn:foo> <urn:bar> <urn:baz>';
+      let select = `select * {<urn:foo> ?p ?o}`;
```
Comment (arb) [inline test/query.spec.js]: Same as above and everywhere else you do this pattern.

File: test/query.spec.js (hunk 1 above)
Comment (snowell) [inline test/query.spec.js]: It is! https://www.w3.org/TR/sparql11-update/#clear

**Pull request merged**